test/runner/parse_metadata_test.dart

```diff
@@ -354,9 +195,9 @@ library foo;
   group("@OnPlatform:", () {
     test("parses a valid annotation", () {
       new File(_path).writeAsStringSync("""
-@OnPlatform(const {
+@OnPlatform({
   'chrome': const Timeout.factor(2),
```

Comment:
nit: consider omitting const for one of the constructors also

Reply:
Can't yet. See the note in the commit message: "Retain const
constructor calls since we can't yet parse the resulting expression if
it is omitted."

And details of why it's hard to support omitting this const at https://github.com/dart-lang/test/issues/915#issuecomment-407923698

Reply:
Ah ok, maybe make a separate test and mark it as skipped then and link to the bug?

Reply:
Done